Templates.xcodeproj/project.pbxproj

```diff
@@ -216,7 +216,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [[ ! $CI ]]; then\n    rake lint:tests\n    rake lint:output\nfi";
+			shellScript = "if [[ ! $CI ]]; then\n  [[ -r ~/.profile ]] && source ~/.profile\n  [[ `which rbenv` ]] && eval \"$(rbenv init - --no-rehash sh)\"\n  if [[ `which bundler` ]]; then\n    bundle exec rake lint:tests\n    bundle exec rake lint:output\n  else\n    echo \"warning: bundler not found, can't lint code. Please install bundler then run bundle install.\"\n  fi\nfi";
```

Comment:
I'm not using `rbenv` so not sure why that would be necessary (if it always is, why isn't it in your `~/.profile` for every new shell?), but if we really need to add that here for brent users, then we should apply that to other repositories too.